runner/operator/argos_bam_operator/v1_0_0/argos_bam_operator.py

```diff
@@ -473,15 +480,14 @@ def get_log_directory(self):
         app = self.get_pipeline_id()
         pipeline = Pipeline.objects.get(id=app)
         output_directory = os.path.join(
-            pipeline.output_directory,
+            pipeline.log_directory,
```

Comment:
why are we setting the output_directory based on the log_directory?

Reply:
It is a `get_log_directory` function. I can rename this variable not to cause confusion.